engines/hdb/sound.cpp

```diff
@@ -1867,40 +1857,33 @@ Audio::AudioStream* Song::createStream(Common::String fileName) {
 	}
 }
 
-void Song::setVolume(int volume) {
-	if (fadingIn)
-		fadeInVol = volume;
-	if (!fadingOut)
-		g_hdb->_mixer->setChannelVolume(handle, volume);
-}
-
-void Song::update() {
-	if (fadingOut) {
-		fadeOutVol = 0;
-		_playing = false;
-		g_hdb->_mixer->stopHandle(handle);
-	}
-	else if (fadingIn) {
-		fadeInVol = g_hdb->_sound->_musicVolume;
-		fadingIn = false;
-	}
-}
+//void Song::update() {
+//	if (fadingOut) {
+//		fadeOutVol = 0;
+//		_playing = false;
+//		g_hdb->_mixer->stopHandle(handle);
+//	}
+//	else if (fadingIn) {
+//		fadeInVol = Audio::Mixer::kMaxChannelVolume;
+//		fadingIn = false;
+//	}
+//}
```

Comment:
Leftover?

Reply:
This is the same case as Sound::updateMusic(). Code that is incomplete, meant to be used for fade in / fade out / cross fade of music tracks. Song::update() is only called in Sound::updateMusic(). 
I think it's better to keep this, in comments, anticipating that someone will complete the implementation in the future -- which by the way shouldn't be that difficult to do, but it should probably be done by someone more familiar with the particular engine code and the disassembly than me. I would do a generic implementation for it, but just use assumptions about the role of "ramp" (whether it's supposed to be a duration of fadein/fadeout in milliseconds) or which points of the game it makes sense to test it etc.